app/static/js/social_links.js

```diff
@@ -0,0 +1,48 @@
+$(window).on('load', () => {
```

Comment:
In this entire file, use either `"` or `'` consistently. We should probably use `"` where possible to math the Python style.